frontend/components/README.md

```diff
@@ -5,7 +5,7 @@ This folder contains MarkBind's [Vue.js](https://vuejs.org/) UI components.
 Most of the styles are based on Bootstrap's markup and CSS, but no dependency on Bootstrap's JavaScript is required. The only required dependencies are:
 
 * [Vue.js](http://vuejs.org/) (required ^v2.x.x, test with v2.5.16).
-* [Bootstrap CSS](http://getbootstrap.com/) (required 4.x.x, test with 4.1.1). VueStrap doesn't depend on a very precise version of Bootstrap.
+* [Bootstrap CSS](http://getbootstrap.com/) (required 4.x.x, test with 4.1.1). Vue c doesn't depend on a very precise version of Bootstrap.
```

Comment:
```suggestion
* [Bootstrap CSS](http://getbootstrap.com/) (required 4.x.x, test with 4.1.1). MarkBind Vue doesn't depend on a very precise version of Bootstrap.
```

Reply:
`MarkBind's Vue components` instead? (since the package is still named `vue-components`)

Reply:
Let's rename the package to `@markbind/markbind-vue`.

Reply:
`@markbind/markbind-vue` sounds a little repititive, I think it should be ok to use a different package name from the `window.xxx` export, I don't think a new developer would mistake that for something else.
Otherwise I could revert it to `VueComponents` for now, and remove the need to reference (`VueComponents.components` - this isn't actually necessary)

Either way don't think we should do it in this pr, it will result in very large diffs reverting/changing what was done in the previous one.

Reply:
Ok, let's go with `@markbind/vue-components` (anyway, we aren't publishing yet) and `window.MarkBindVue` for now.

Well-known packages for reference:
- https://www.npmjs.com/package/@angular/material
- https://www.npmjs.com/package/@ng-bootstrap/ng-bootstrap

Reply:
or we could just do away with scoped packages, since `MarkBindVue` is pretty unique. Also saves having to modify the existing ones (`markbind/markbind-cli`) for consistency. Gatsby seems to practice this as well. 🤔

But we can get to that when there's a need to publish any subpackages separately as you mentioned